app/helpers/hosts_helper.rb

```diff
@@ -270,7 +270,7 @@ def show_templates
       content_tag(:th, _("Template Type")) + content_tag(:th) +
       templates.map do |kind, tmplt|
         content_tag :tr do
-          content_tag(:td, _("%s Template") % kind) +
+          content_tag(:td,  _("%s Template") % kind) +
```

Comment:
you added a space but did not extract the kind for translation. Why not move the template part also to the pretty name?